app/Domain/Users/Controllers/EditOwn.php

```diff
@@ -254,5 +280,29 @@ public function post(): void
             //Redirect
             FrontcontrollerCore::redirect(BASE_URL . "/users/editOwn" . $tab);
         }
+
+        /**
+         * Returns list of supported varying date-time formats.
+         * @link https://www.php.net/manual/en/class.datetimeinterface.php#datetimeinterface.constants.types
+         * 
+         * @return [<string,string>] Format of ID => date-time string
+         */
+        private function getSupportedDateTimeFormats(): array
+        {
+            return [
+                'dates' => [
```

Comment:
I created these originally from PHP's `DateTimeInterface` constants, as there can be a long list of possibles.  I think at some point we need to let the timezone be able to control this, or we set the locale somehow in PHP if possible.

Unless you know of a good way to represent all possible format values in a list without being overwhelming for the user.

An aside thought while I was typing this too would be to allow a plugin (once that feature is good to go) that allows extending this based on the user base the installer has (i.e.: a company with only American employees probably won't care as much about formats fit for Germany and vice-versa).

Reply:
In any case we need the german date time format in the list since a large number of our users are from german speaking countries. The alternative to this would be to just make it an input and do a dateTime check when the value is submitted: 
```
$dt = DateTime::createFromFormat(FORMAT time());

$dt !== false && !array_sum($dt::getLastErrors());
```
If the result is false it's not a valid format.

Reply:
Do you have a list of the top 3-5 countries for your user base?

Having worked at a company before that allowed their end-users to input arbitrary values for date-time it got ugly.  If we want to expand beyond what PHP uses as constants, then I sense it would be better to add as needed.  Otherwise you then need to also educate the user who may want a different format but isn't very tech-savvy to try and understand how to format a PHP date-time string.

Reply:
What I did for this is go through the list of countries in that wiki article and pick out some of the common formats on top of the one you mentioned.  Due to no longer depending heavily on the `DateTimeInterface` constants like I was doing at some point (prior to submitting this PR) I realized I'm just saving the format and not some sort of identifier.  So that made this refactor much easier.

However, due to the extra text space required for some of the formats, I did have to expand the width a bit on the `ticketDetails.sub.php` template.

Reply:
Thank you this is great!